src/data/markdown/docs/03 cloud/05 Project and Team Management/04 SAML SSO.md

```diff
@@ -66,3 +70,17 @@ As you can see below, the default project is already selected:
 3. From the drop-down you can choose the default project(s) newly provisioned users will be assigned, then click on update button:
 ![SAML SSO update default projects](images/04-SAML-SSO/saml-sso-update-default-project.png)
 4. Now the users assigned to the IdP application can sign in to k6 Cloud and they'll be automatically added to the specified project(s) in step 3 with __project read/write__ permissions.
+
+## Fine-grained users to projects assignment using teams
```

Comment:
This title is not very clear to me ... how about `Auto-assigning your internal teams to k6 projects`?